extension/src/test/suite/experiments/index.test.ts

```diff
@@ -997,21 +989,22 @@ suite('Experiments Test Suite', () => {
     }).timeout(WEBVIEW_TEST_TIMEOUT)
 
     it('should be able to handle a message to move a column group to the start of the table', async () => {
-      const { experiments, mockMessageReceived, messageSpy } =
-        await buildExperimentsWebview({
+      const { mockMessageReceived, messageSpy } = await buildExperimentsWebview(
+        {
           disposer: disposable
-        })
+        }
+      )
 
       messageSpy.resetHistory()
 
       const movedGroup = 'params:params.yaml'
 
-      const tableChangePromise = experimentsUpdatedEvent(experiments)
+      const messageSent = waitForSpyCall(messageSpy, messageSpy.callCount)
```

Comment:
[F] I had to add this new util as the data for the message is now gathered async.

Reply:
Where possible we still rely on the old util (experimentsUpdatedEvent(experiments)`)